scripts/preprocess.sh

```diff
@@ -34,12 +34,50 @@ if [[ "${CF_INSTANCE_INDEX}" = "0" ]]; then
   fi
 
   # Import the config from sync dir
-  drush config-import -y
+  config_status_file=$(mktemp)
+
+  CONFIG_STATUS=$(drush config-status &>$config_status_file)
+
+  printf "%s\n" "$CONFIG_STATUS"
```

Comment:
Minor thing - I think it's easier not using printf and just using echo "${CONFIG_STATUS}" - is there a reason to use `printf`?

Reply:
Only to maintain the formatting of the `drush config-status` output. Otherwise it comes out as a string.